src/components/ProgressBarContainer.ts

```diff
@@ -81,7 +87,7 @@ export default class ProgressBarContainer extends Component<ProgressBarContainer
             className: this.props.class,
             colorSwitch: this.props.textColorSwitch,
             maximumValue: this.state.maximumValue,
-            onClickAction: (this.props.onClickMicroflow || this.props.onClickPage) && this.props.mxObject
+            onClickAction: (this.props.onClickMicroflow || this.props.onClickNanoflow || this.props.onClickPage) && this.props.mxObject
```

Comment:
Should compare with `this.props.onClickOption !== "doNothing"`

Reply:
Done